### PR TITLE
Add config/auth/latex docs

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: docs/source/installation/config.rst [hunk 1]
```diff
@@ -0,0 +1,781 @@
...
+.. data:: PROFILE
+
+    Enables the Python profiler.  The profiler output is stored in
+    ``<TEMP_DIR>/*.prof``.
+
+    Default: ``False``
+
+.. data:: SMTP_USE_CELERY
+
+    If disabled, emails will be send immediately insteaad of being
```
Comment (pferreir) [inline docs/source/installation/config.rst]: > insteaad

File: docs/source/installation/config.rst [hunk 2]
```diff
@@ -0,0 +1,781 @@
...
+
+.. data:: FLOWER_URL
+
+    The URL of the `Flower`_ instance monitoring your Celery workers.
+    If set, a link to it will be displayed in the admin area.
+
+    To use it flower, install it using ``pip install flower``, then
+    start it using ``indico celery flower``. By default it will listen
+    on the same host as specified in :data:`BASE_URL` (plain HTTP) on port
+    5555.  Authentication is done using OAuth so only indico administrators
```
Comment (mvidalgarcia) [inline docs/source/installation/config.rst]: > **I**ndico

File: docs/source/installation/config.rst [hunk 3]
```diff
@@ -0,0 +1,781 @@
...
+
+.. data:: HELP_URL
+
+    The URL used for the "Help" link in the footer.
+
+    Default: ``'https://learn.getindico.io'``
+
+.. data:: LOGO_URL
+
+    The URL to a custom logo.  If unset, the default indico logo is used.
```
Comment (mvidalgarcia) [inline docs/source/installation/config.rst]: > **I**ndico

File: docs/source/installation/config.rst [hunk 4]
```diff
@@ -0,0 +1,781 @@
...
+    Default: ``{}``
+
+
+Logging
+-------
+
+.. data:: LOGGING_CONFIG_FILE
+
+    The path to the logging config file.  Unless an absolute path is specified,
+    the path is relative to the location of the indico config file after
```
Comment (mvidalgarcia) [inline docs/source/installation/config.rst]: > **I**ndico

File: docs/source/installation/config.rst [hunk 5]
```diff
@@ -0,0 +1,781 @@
...
+    ``<TEMP_DIR>/*.prof``.
+
+    Default: ``False``
+
+.. data:: SMTP_USE_CELERY
+
+    If disabled, emails will be send immediately instead of being
+    handed to a Celery background worker.  This is often more convenient
+    during development as you do not need to run a Celery worker while still
+    receiving emails sent from indico.
```
Comment (mvidalgarcia) [inline docs/source/installation/config.rst]: > **I**ndico

File: docs/source/installation/config.rst [hunk 6]
```diff
@@ -0,0 +1,781 @@
...
+    Disabling it may result in emails not being sent if the mail server is
+    unavailable or some other failure happens during email sending.  Because
+    of this, the setting should never be disabled in a production environment.
+
+    Default: ``True``
+
+.. data:: COMMUNITY_HUB_URL
+
+    The URL of the community hub. This should only be changed when using a local
+    instance of Mereswine to debug the interface between indico and Mereswine.
```
Comment (mvidalgarcia) [inline docs/source/installation/config.rst]: > **I**ndico

File: docs/source/installation/config.rst [hunk 7]
```diff
@@ -0,0 +1,781 @@
...
+
+    Default: ``xelatex``
+
+.. data:: STRICT_LATEX
+
+    Enables strict mode for LaTeX rendering, in which case a non-zero
+    status code is considered failure.
+
+    LaTeX is rather generous when it comes to using a non-zero exit code.
+    For example, having an oversized image in an abstract # is enough to
```
[the remark this answers is not in the document]
Comment (ThiefMaster) [reply docs/source/installation/config.rst]: copy&paste screwup from the example config

File: docs/source/installation/config.rst [hunk 8]
```diff
@@ -0,0 +1,781 @@
...
+.. data:: PROFILE
+
+    Enables the Python profiler.  The profiler output is stored in
+    ``<TEMP_DIR>/*.prof``.
+
+    Default: ``False``
+
+.. data:: SMTP_USE_CELERY
+
+    If disabled, emails will be send immediately instead of being
```
Comment (mic4ael) [inline docs/source/installation/config.rst]: will be sent is the correct form

File: docs/source/installation/config.rst [hunk 9]
```diff
@@ -0,0 +1,781 @@
...
+    - ``X-Forwarded-Host`` -- the hostname as specified in :data:`BASE_URL` (can
+      be omitted if the ``Host`` header is correct)
+
+    .. warning::
+
+        This setting **MUST NOT** be enabled if the server is
+        accessible directly by untrusted clients without going through
+        the proxy or users will be able to spoof their IP address by
+        sending a custom ``X-Forwarded-For`` header.  You need to
+        configure your firewall so only requests coming your proxy or load
```
Comment (mic4ael) [inline docs/source/installation/config.rst]: coming your proxy?

File: docs/source/installation/config.rst (hunk 8 above)
Comment (ThiefMaster) [reply docs/source/installation/config.rst]: typo :p

File: docs/source/config/auth.rst [hunk 10]
```diff
@@ -0,0 +1,284 @@
...
+
+It must be set to a dict mapping a unique (internal) name of the auth
+provider (e.g. ``mycompany-ldap``) to a dict of whatever data is
+needed for the given provider.
+
+The following keys are available in the provider data:
+
+- ``type`` -- **Required.** The type of the provider. Valid values
+  are e.g. ``ldap``, ``oauth``, ``shibboleth``, and whatever custom
+  providers you have installed
```
Comment (mvidalgarcia) [inline docs/source/config/auth.rst]: Missing ending dot.

**Pull request merged**